### ISSUE TITLE
About the dimension of re-id feature

### ISSUE THREAD

**Issue by 3bobo** [ifzhang/FairMOT#177]
Hi, thanks for your work! Here is my question. I noticed that in the code the default dimension of re-id feature is 512:
https://github.com/ifzhang/FairMOT/blob/ca78d3f01d1953c3d72ee8b01f24a5ddec2689c3/src/lib/opts.py#L142
and the train and track files also do not change it to 128-dim.
However, your paper claims that 128 dimension feature has a better performance. So I should change it to 128 if I want to get the similar results as your paper, or the 512 dimension feature could also get the performance?

**Comment by ifzhang** [comment]
On our validation set 128 dim performs better than 512. I do not test them on the test set. The paper's results on test set using 512-dim and I put the model on github. Maybe 128-dim can perform better on the test set, I'm not sure.

**Comment by khodabakhshih** [comment]
Pretty good question.
The dimension of Re-ID vector is about 20000. Would you please provide us some interpretations about why reid_dim=128 or 512 performs better than higher dimensional vectors?
